src/App.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { useContext } from "react";
+import "./App.css";
+import { AppContext } from "./components/AppContext/AppContext";
+
+export const App = () => {
+  const { data } = useContext(AppContext);
+
+  const images = data.map(({ image, name, id }) => (
```

Comment:
Do you think it would be overkill to do some checking here for empty/loading states as well?

Reply:
`AssetFetcher` prevents this from rendering until the data is fetched